annotate.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.name = "annotate"
   s.version = Annotate.version
 
-  s.required_ruby_version = '>= 1.9.3'
+  s.required_ruby_version = '>= 2.2.5'
```

Comment:
It appears this is the breakage https://github.com/guard/listen/commit/c65339c931c453848ed637b15edba5a064065d05